include/openenclave/internal/sgxtypes.h

```diff
@@ -554,7 +554,7 @@ oe_thread_data_t* oe_get_thread_data(void);
 
 #define TD_MAGIC 0xc90afe906c5d19a3
 
-#define OE_THREAD_LOCAL_SPACE (3304)
+#define OE_THREAD_LOCAL_SPACE (3816)
```

Comment:
Are we also planning to do the same for linux_errno?

Reply:
I wasn't planning to originally, since `linux_errno` is used in the corelibc implementation of `errno.c` (rather than only used in oelibc), but I could do that too for data hygiene. I'll take a look.